packages/ajna-contracts/scripts/common/hardhat.utils.ts

```diff
@@ -224,3 +225,7 @@ export class HardhatUtils {
     }
   }
 }
+
+export function logGasUsage(receipt: ContractReceipt, methodName: string) {
```

Comment:
This seems like a nice addition, could you mention it in the PR description so other devs are aware?

Reply:
done :)